ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/IndividualPages/HelpTheCourtDecide.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ServiceWebsite.AcceptanceTests.Helpers;
+
+namespace ServiceWebsite.AcceptanceTests.Pages.IndividualPages
+{
+    public class HelpTheCourtDecide
+    {
+        private readonly BrowserContext _context;
+        private readonly CommonPages _commonPages;
+
+        public HelpTheCourtDecide(BrowserContext browserContext, CommonPages commonPages)
+        {
+            _context = browserContext;
+            _commonPages = commonPages;
+        }
+
+        public void IndividualHelpsCourtDecide()
```

Comment:
Do we need these empty methods

Reply:
![image](https://user-images.githubusercontent.com/36300564/57511204-a3fd6e00-7300-11e9-8574-d22062fadc02.png)